nupic/research/frameworks/pytorch/imagenet/imagenet_tune.py

```diff
@@ -284,6 +284,9 @@ def _setup(self, config):
             worker_config["rank"] = i
             status.append(w.setup_experiment.remote(worker_config))
 
+        if config.get("checkpoint_at_init", False):
+            self.save()
```

Comment:
Which directory does this get saved in?

Reply:
When `save` is called without `checkpoint_dir`, `tune` will place the checkpoint in the results dir postfixed with the "epoch" just like any other checkpoint.

Reply:
Ray is training the models asynchronously. There is no guarantee the `setup_experiment` call finished at this point.
You should move the `save()` to after `ray.get(status)`